Sources/TuistSupport/Constants.swift

```diff
@@ -60,17 +60,31 @@ public enum Constants {
     /// These variables are not supposed to be used by end users
     /// But only eg. for acceptance tests and other cases needed internally
     public enum EnvironmentVariables {
-        public static let verbose = "TUIST_VERBOSE"
-        public static let colouredOutput = "TUIST_COLOURED_OUTPUT"
-        public static let versionsDirectory = "TUIST_VERSIONS_DIRECTORY"
+        public static let verbose = "TUIST_CONFIG_VERBOSE"
+        public static let colouredOutput = "TUIST_CONFIG_COLOURED_OUTPUT"
+        public static let versionsDirectory = "TUIST_CONFIG_VERSIONS_DIRECTORY"
         @available(*, deprecated, message: "This is used only for acceptance tests, see https://github.com/tuist/tuist/issues/2777")
```

Comment:
@pepibumur not related to this PR, but if we don't plan to do anything for this, can we remove this deprecation, and maybe made it clear from the name that it shouldn't be used for other purposes? IMO it's a bit annoying to always have that warning popping up knowing we can't fix it

Reply:
Agree. Let's remove it.